.gitignore

```diff
@@ -21,6 +21,10 @@ _testmain.go
 
 *.exe
 
+# Backup files
+*~
+
```

Comment:
I think ignores that should apply to all repositories belong in `~/.gitignore`; it is why I am not including vim swap files in this .gitignore.